hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/FlinkWriteClients.java

```diff
@@ -213,6 +213,7 @@ public static HoodieWriteConfig getHoodieClientConfig(
                 .parquetBlockSize(conf.getInteger(FlinkOptions.WRITE_PARQUET_BLOCK_SIZE) * 1024 * 1024)
                 .parquetPageSize(conf.getInteger(FlinkOptions.WRITE_PARQUET_PAGE_SIZE) * 1024 * 1024)
                 .parquetMaxFileSize(conf.getInteger(FlinkOptions.WRITE_PARQUET_MAX_FILE_SIZE) * 1024 * 1024L)
+                .withWriteUtcTimezone(conf.get(FlinkOptions.WRITE_UTC_TIMEZONE))
```

Comment:
Can this function be Flink specific now?

Reply:
Firstly, It's a storage conf to control value of TimeStamp type written to parquet file. If not set into the storage conf, there would be  many changes to the write path just for passing the UTC option into the underlying writer.

Reply:
+1 to @danny0405 . Hudi now has a lot of options, I would prefer to make this option Flink specific.